charts/alfresco-sync-service/values.yaml

```diff
@@ -55,11 +55,28 @@ repository:
 # user: username to authenticate as.
 # password: credential to use to authenticate to the broker.
 messageBroker:
-  url: >-
-    failover:(nio://acs-activemq-broker:61616)?timeout=3000&jms.useCompression=true
-  user: admin
-  password: admin
-  existingSecretName: null
+  enabled: false
+  external:
+    url:
+    # failover:(nio://acs-activemq-broker:61616)?timeout=3000&jms.useCompression=true
+    user: alfresco
+    password: admin
+    existingSecretName:
```

Comment:
I am not sure about having extraneous values in a section of a dependency, that could have unexpected behavior in case of conflicts. And for what? just to have a single section in the values for handling everything related to an infra component (internal or external) - may not be bad for the user but now I am concerned about the consistency of this approach (we really need to document this soft design choice somewhere or we will endup doing inconsistent things with time passing)

Reply:
That is very true.
I quickly created https://alfresco.atlassian.net/browse/OPSEXP-2043 to track that

Reply:
If you don't mind, I will kepp this one for documentation purpose as it's not linked to another chart and remove the ones which are documented as part of the dependent charts